### DISCUSSION TITLE
aideck antenna

### DISCUSSION THREAD

**Discussion by 4perros** [bitcraze/discussions#916] [Q&A]
hello marcus
i noticed that my aideck has lost its antenna. do you know where i could buy a new one?
regards
rodrigo calvo

**Comment by krichardsson** [comment]
Unfortunately we do not know if it is possible to buy new antennas. Maybe you can check with [ublox](https://www.u-blox.com/) that makes the NINA-W102 module?

**Reply by 4perros** [reply]
hello
thank you for your answer. I have contacted ublox as suggested.I got an answer saying that ublox does not sell the part (antenna). Do you have any ideas as to how to fix the problem or is it possible to have the aideck replaced
Regards
Rodrigo

**Reply by tobbeanton** [reply]
The [PRO-OB-607](https://www.digikey.se/sv/products/detail/abracon-llc/PRO-OB-607/13174394?s=N4IgTCBcDaIAoCUDyBaJAhFA2ADAdhAF0BfIA) looks a lot like it.

**Reply by 4perros** [reply]
hello
thank you for your quick answer. Yes i was actually trying to figure out the part number and found a link to element14 (electronics) in which they have that precise antenna. So i will buy that one and see if it works, i will keep you inform. Thanks a lot for the info.
Regards
rodrigo

**Reply by 4perros** [reply]
hello
i purchased the PRO OB 607 antenna, placed it on the NINA and it fits ok unfortunately it is too low,
it sits flash with the metal box, so i don’t think it is suitable i included a photo to show you what i mean.
I don’t know what else i could do. 
Does the aideck NINA antenna has a part number stamped on it?
regards
rodrigo

![IMG_0282](https://github.com/bitcraze/discussions/assets/13658763/334794a1-f2c8-4ef1-b347-cae764ec9cf5)

**Reply by tobbeanton** [reply]
That is to bad, I guess UBLOX has a custom made version which is higher. Have you tried the connection? We will buy some and do some reflective antenna tests with, perhaps it works quite well anyway. The antenna should not be in contact with the metal shield though.

**Reply by tobbeanton** [reply]
> Does the aideck NINA antenna has a part number stamped on it?

No part-number stamped on it

**Reply by 4perros** [reply]
hi
No i haven’t tried the connection yet i have to solder the antenna. i am waiting for some solder paste, might be able to do it on the weekend.
will keep you posted. Let me know how the tests go please.
regards
rodrigo

**Reply by tobbeanton** [reply]
I did some reflection tests on the [PRO-OB-607](https://www.digikey.se/sv/products/detail/abracon-llc/PRO-OB-607/13174394?s=N4IgTCBcDaIAoCUDyBaJAhFA2ADAdhAF0BfIA) antenna using our mini-VNA.

First thing it does not fit exactly. It is a bit wider and less high, however it is close enough so that is can be soldered there. Secondly it is not as good as the original but should work, even if the antenna touches the shield (touching it or not does not make a big difference).

Original:
![NINA-original](https://github.com/bitcraze/discussions/assets/6593186/12e9992b-b675-476e-ab6f-142ba9136203)


PRO-OB-607-antenna_without_shied:
![NINA-PRO-OB-607-antenna_without_shied](https://github.com/bitcraze/discussions/assets/6593186/279f86e1-c3f8-4f22-b9b4-ee390034e548)
![image](https://github.com/bitcraze/discussions/assets/6593186/ed93b6f6-3b0f-411c-8712-3f15e6acfab7)


PRO-OB-607-antenna_with_shied_touching_antenna:
![NINA-PRO-OB-607-antenna_with_shied_touching_antenna](https://github.com/bitcraze/discussions/assets/6593186/bb36bb3a-3d83-4ea1-81b4-7db17cffb69e)
![image](https://github.com/bitcraze/discussions/assets/6593186/3b961574-25da-4e81-951c-96bef5bb9944)

**Reply by 4perros** [reply]
hello
thank you very much for all the information, its very informative. I will let you know the outcome. thank you again.
regards
rodrigo

**Reply by 4perros** [reply]
hi
how did you hold antenna in place while you soldered it?
regards
rodrigo

**Reply by tobbeanton** [reply]
I used hot air and a pair of tweezers but I guess a small soldering tip would do as well.

**Reply by 4perros** [reply]
hi
thank you for all the information. I managed to solder aerial with soldering iron and solder paste. 
regarding the new battery (350mAh) is that going to be available in stores in Australia? 
regards
rodrigo

**Reply by knmcguire** [reply]
@4perros that's a different topic so it's usually best to make a new discussion. But I'll make an exception.

The [new batteries](https://store.bitcraze.io/collections/spare-parts/products/350mah-lipo-battery) initially will be available at [our store](https://store.bitcraze.io/) and will slowly trickle down to the [other vendors in other countries](https://www.bitcraze.io/buy/local-retailers/). We usually don't pressure them to have our full offering, but we do try to notify them every time they place a new order. You can also ask their support if they are planning to have those in stock to add a bit of pressure 😄 

btw, don't forget to mark an answer to the discussion to close it off properly.

**Reply by 4perros** [reply]
hi
thank you for the information.
regards

**Comment by docckey** [comment]
Sorry to bother you, but I have also encountered the same problem (antenna loss). How did you solve it?

**Reply by gemenerik** [reply]
It's unfortunate that you lost your antenna! As you can see from the discussion above, a similar part was purchased as a replacement, but it seems it's not a perfect match. Unfortunately, the original part isn't available for separate purchase. The purchased antenna differs slightly from the original, causing it to touch the shielding, though it's not considered a major issue. Soldering it into place isn't very easy, but it is possible. Therefore, I would recommend purchasing it and giving it a try.

**Reply by docckey** [reply]
Can you help me check if the lost part is the antenna?
![1A1511737FE671A6F222FF5B31E0A10C](https://github.com/bitcraze/discussions/assets/148048159/60ef7a1f-caab-4a1d-9005-ff6a7426e4cd)

**Reply by gemenerik** [reply]
Yes it is missing the antenna, see my comment on #1259.

Can you please keep the discussion here about replacing the antenna, and the one in #1259 about identifying the missing part and what it implies?
